packages/client/src/v2-events/features/events/actions/quick-actions/register.ts

```diff
@@ -21,7 +21,7 @@ export const register: QuickActionConfig = {
     }
   },
   onConfirm: ({ event, actions, customActions }) => {
-    if (event.status === EventStatus.enum.DECLARED) {
+    if (!event.flags.includes('validated')) {
```

Comment:
This is a temporary measure that is needed as long as validate is a core action, and not a custom action

We plan on changing it to a custom action later on, at which point this should be removed.